src/gluonts/evaluation/_base.py

```diff
@@ -461,6 +466,25 @@ def get_aggregate_metrics(
                 for q in self.quantiles
             ]
         )
+
+        # Compute OWA if required
+        if self.calculate_owa:
+            if totals["sMAPE_naive2"] == 0 or totals["MASE_naive2"] == 0:
+                warnings.warn(
+                    "OWA cannot be computed as Naive2 yields an sMAPE or MASE of 0."
+                )
```

Comment:
Elsewhere we use the `logging` module for warnings, would it make sense to also use it here?